src/factory.ts

```diff
@@ -54,10 +78,19 @@ export default class LuaFactory {
         luaWasm.module.FS.writeFile(path, content)
     }
 
-    public async createEngine(options: ConstructorParameters<typeof LuaEngine>[1] = {}): Promise<LuaEngine> {
+    /**
+     * Creates a Lua engine with the specified options.
+     * @param [options] - Configuration options for the Lua engine.
+     * @returns - A Promise that resolves to a new LuaEngine instance.
+     */
+    public async createEngine(options: CreateEngineOptions = {}): Promise<LuaEngine> {
         return new LuaEngine(await this.getLuaModule(), options)
     }
 
+    /**
+     * Gets the Lua WebAssembly module.
```

Comment:
would be nice to explain how this is useful, like access the lua api directly, etc